### ISSUE TITLE
Func with schema qualified names not working properly after upgrade from 8 to 10

### ISSUE THREAD

**Issue by andrwo** [vitaly-t/pg-promise#765]
I just upgraded my pg-promise from 8.7 to latest, and the db.func() stopped working.

This is possibly related to issue #671 that caused this.

My code is simply:  
```
const db_rec = await db.func("svc.sf_event_create", [5, 500, 1000, 46, 1, 23, 46]);
```
I am hitting error:
error: `function svc.sf_event_create(integer, integer, integer, integer, integer, integer, integer) does not exist`

query: `select * from "svc.sf_event_create"(5,500,1000,46,1,23,46)`

From the monitor query: it seems like the quote marks " " added to the function name + schema scope are the offending issue. However I can't remove the schema scoping "svc." from my queries, and db.func's first parameter is expecting a string.

Running this on DBeaver gives the same error, until the quote marks are removed. 

### Environment

* Version of pg-promise: 10.7.3
* OS type (Linux/Windows/Mac): Windows
* Version of Node.js: 14.15.0

Workarounds:

Change query to: 
```
const db_rec = await db.query("SELECT * FROM svc.sf_event_create($1:csv)", [[5, 500, 1000, 46, 1, 23, 46]]);
```

**Comment by vitaly-t** [comment]
You can add access to `svc` schema, via initialization option `schema`:

```js
const initOptions = {
    schema: ['public', 'svc']
}
```

and then omit use of schema name from the function name.

**Comment by vitaly-t** [comment]
After a bit of thinking, I decided to amend filter `alias` to support auto name split, based on `.`

**Comment by andrwo** [comment]
Thanks @vitaly-t for the quick turnaround, I also think it will be best to handle the alias property to avoid ambiguities in resolution.

Or maybe some solution on the first parameter that optionally takes care of schema, that is similar to what you did for the TableName helper (https://vitaly-t.github.io/pg-promise/helpers.TableName.html).

**Comment by vitaly-t** [comment]
I have published [10.8.0](https://github.com/vitaly-t/pg-promise/releases/tag/10.8.0), which extends [Alias Filter] to auto-split SQL name (based on `.`), to support composite SQL names.

[Alias Filter]:https://github.com/vitaly-t/pg-promise#alias-filter

**Comment by andrwo** [comment]
Yes, I am happy, thanks!
